pkg/config/services.go

```diff
@@ -353,7 +353,7 @@ var SupportedServices = serviceConfigs{
 		DimensionRegexps: []*regexp.Regexp{
 			regexp.MustCompile("destinationEdge/(?P<DestinationEdge>[^/]+)"),
 			regexp.MustCompile("accelerator/(?P<Accelerator>[^/]+)"),
-			regexp.MustCompile("endpointGroup/(?P<EndpointGroup>[^/]+)"),
+			regexp.MustCompile("endpoint-group/(?P<EndpointGroup>[^/]+)"),
```

Comment:
According to AWS [docs](https://docs.aws.amazon.com/global-accelerator/latest/api/API_DescribeEndpointGroup.html), GA endpoint group ARNs use that format. Example:
```
aws globalaccelerator describe-endpoint-group
    --endpoint-group-arn arn:aws:globalaccelerator::012345678901:accelerator/1234abcd-abcd-1234-abcd-1234abcdefgh/listener/6789vxyz-vxyz-6789-vxyz-6789lmnopqrs/endpoint-group/ab88888example 
```